checker/plugins/firejail.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import annotations
+
+import os
+
+from .scripts import RunScriptPlugin
+
+
+PATH = 'PATH'
+PYTHONPATH = 'PYTHONPATH'
+
+
+class SafeRunScriptPlugin(RunScriptPlugin):
+    """Plugin for running scripts in safe mode."""
+
+    name = "safe_run_script"
+
+    Args = RunScriptPlugin.Args
+
+    # TODO(uniqum): at the moment "--queit" option of firejail may stil put extra strings in the output
+    def _run(self, args: Args, *, verbose: bool = False) -> str:
```

Comment:
Seems the problem we faced in tests can be tackled here - you need to check jirejail exists and through clear error msg